lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -46,36 +55,33 @@ class Meta:
         ]
 
 
+class ScoreField(serializers.Field):
```

Comment:
A Serializer's `to_representation` method isn't allowed to return None, apparently, but a Field's can!

Reply:
also docstring